### ISSUE TITLE
[MAYA-111691] Support for card drawMode with textures in Viewport2.0

### ISSUE THREAD

**Issue by radon199** [Autodesk/maya-usd#1382]
**Is your feature request related to a problem? Please describe.**
Viewport 2.0 does not support the UsdGeom.modelAPI card displayMode with textures. Even if the texture attributes are assigned the card will draw as a default cross.
I think this is due to the fact that card draw mode is done with a custom material, and that material has not been translated into a Maya fragment graph like UsdPreviewSurface and the other related nodes have been.
We use this feature in production to remove the cost of drawing complex assemblies by instead drawing them as a single rendered card.
Can we have this feature of Hydra supported?

**Comment by radon199** [comment]
The custom material I was mentioning is this, which UsdImagingGL will assign when processing the draw mode : 
https://github.com/PixarAnimationStudios/USD/blob/release/pxr/usdImaging/usdImagingGL/shaders/drawMode.glslfx

**Comment by santosd** [comment]
Hello @radon199 , Thank you for sending a link to the material. Wondering if you could send us a usd file that is setup to reproduce the issue so we can take a look at this one. Thank you.

**Comment by radon199** [comment]
I have something and am just waiting on approval to add it here.

**Comment by santosd** [comment]
Hi @radon199, am I missing something? Here are the results for me in VP2 and Hydra:

VP2
![image](https://user-images.githubusercontent.com/29235905/118135746-617c3980-b3d1-11eb-8090-9f3805748e08.png)

MtoH Hydra:
![image](https://user-images.githubusercontent.com/29235905/118135926-8d97ba80-b3d1-11eb-83f7-8041c4e9c6ce.png)

I also opened the file in USD view and still don't get anything on the cards.

**Comment by radon199** [comment]
@santosd Sorry, I forgot to make the image file paths relative in the asset usd. Please see the following updated set of data that hopefully will work. I tested it in usdview on a second machine.

[DisplayCardExample.zip](https://github.com/Autodesk/maya-usd/files/6474472/DisplayCardExample.zip)

**Comment by santosd** [comment]
@radon199, Thank you for the file. I should have caught that the paths were not relative, my bad. I spoke with the team and will be logging this one internally so that we can take a look into adding this support into the VP2 render delegate.

**Comment by dj-mcg** [comment]
FWIW, this is currently one of the last remaining blockers for adoption at Pixar. Thanks for having a look!

**Comment by williamkrick** [comment]
@radon199 would it be alright with you if I use your DisplayCardExample scene as the basis for an automated test in MayaUSD?

**Comment by radon199** [comment]
@williamkrick From my side (ie what is setting card mode and the textures) it is fine, only thing to note was the fridge asset itself is from the Pixar kitchen dataset.

**Comment by dj-mcg** [comment]
Hey folks, I'm asking about usage of the fridge asset with the USD team and Pixar legal right now. I'll hopefully have an answer for you shortly

**Comment by williamkrick** [comment]
Great I will use the existing asset. Thanks!

**Comment by dj-mcg** [comment]
Hi folks - I'm still not seeing the card images when opening the attached usda file when rendering with the vp2RenderDelegate despite this issue being closed a long time ago. I'm using Maya 2023.3 and MayaUsd 0.25.0. I'm on linux

**Comment by santosd** [comment]
Hi @dj-mcg , looks like I validated this in Maya 2023, when I originally tested and I tried again today in Maya 2024 with 0.25.0 so it should be working on your end. Once thing to note is that we also added support for texture mode in Maya 2023.3. I know by default texture mode is turned off so in order to see the textures on the cards you would need to enable texture mode. Let me know if that helps, if not, I will take a look at 2023.3 and see if I see the issue on my end.
